community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseBuilder.java

```diff
@@ -118,6 +119,16 @@ public GraphDatabaseBuilder setConfig( Map<String, String> config )
     }
 
     /**
+     * Retrieve the configured configuration values.
+     *
+     * @return the configured configuration values.
+     */
+    public Config config()
+    {
+        return new Config( config );
+    }
```

Comment:
We need to access the config from the builder to find and disable all configured (bolt) connectors.

**Question:** does creating a `Config` object like this, without passing in any settings classes, work as expected? (it works for our use case, but since this is part of public API, other usages will occur...)